src/routes/popup/receive.tsx

```diff
@@ -70,7 +70,7 @@ export function ReceiveView({ walletName, walletAddress }: ReceiveViewProps) {
           </Text>
           <QRCodeWrapper>
             <QRCodeSVG
-              fgColor="#fff"
+              fgColor="#000"
```

Comment:
```suggestion
              fgColor="#fff"
```
The receive QR doesn't need updating. Only the dynamic QR from "Generate QR Code" when you are exporting your wallet